web/src/OverviewActionBar.tsx

```diff
@@ -224,6 +245,43 @@ export let ButtonRightPill = styled(ButtonRoot)`
   border-radius: 0 4px 4px 0;
 `
 
+const FilterTermTextField = styled(TextField)`
+  & .MuiOutlinedInput-root {
+    border-radius: ${SizeUnit(0.125)};
+    border: 1px solid ${Color.grayLighter};
+    background-color: ${Color.gray};
+    transition: border-color ${AnimDuration.default} ease;
+
+    &:hover {
```

Comment:
The border colors here are inconsistent with the resource name filter in the sidebar. I don't know if one of us failed to follow the design or the design didn't specify or the design is inconsistent. Would you mind tracking that down? (We should probably make a component that standardizes these so that we don't have to think about it, but that doesn't need to be in this PR)

Reply:
Surbhi's design has this input styled like the other filter buttons (which have less-rounded corners and blue borders), rather than the resource filter field. Let me know if that doesn't resolve your question!